content/agent/faq/upgrade-to-agent-v6.md

```diff
@@ -0,0 +1,405 @@
+---
+title: Upgrade to Datadog Agent v6
+kind: faq
+---
+
+<div class="alert alert-info">
+To do a fresh install of Datadog Agent v6, refer to the Agent <a href="https://app.datadoghq.com/account/settings#agent">installation process</a>.
+</div>
+
+## Amazon Linux
+
+A script is available to automatically install or upgrade to the new Agent. It sets up the package repositories and installs the Agent package for you. When upgrading, the import tool also searches for an existing `datadog.conf` from a prior version, and converts Agent and Check configurations according to the new v6 format.
+
+### One-step Upgrade
+
+The Agent v6 installer can automatically convert v5 configurations during the upgrade:
```

Comment:
applies to all OSes: I feel this line is redundant with the OS-level intro above. I'd suggest removing it, and replacing it with something like: 

```
`DD_UPGRADE=true` makes the installer automatically convert v5 configurations during the upgrade
```

after the shell command line.

WDYT?

Reply:
I agree, but here the idea is the support can send this page with the correct anchor to a customer and the customer only have to cp/cv the one line script. I'm going to add the API key though.